packit_service/worker/handlers/github_handlers.py

```diff
@@ -672,29 +671,13 @@ def __init__(self, config: ServiceConfig, event: IssueCommentEvent):
         self.config = config
         self.event = event
         self.project = self.event.get_project()
-        # Get the latest tag release
-        self.event.tag_name = self.get_tag_name()
         self.package_config: PackageConfig = self.get_package_config_from_repo(
             self.project, self.event.tag_name
         )
         if not self.package_config:
             raise ValueError(f"No config file found in {self.project.full_repo_name}")
         self.package_config.upstream_project_url = event.project_url
 
-    def get_tag_name(self) -> Optional[str]:
-        """
-        Get the tag name of the latest upstream release.
-
-        :return: tag name
-        """
-        try:
-            tag_name = self.project.get_latest_release().tag_name
```

Comment:
What about  adding `allow_prereleases=False` argument to `get_latest_release` in ogr?

Reply:
I am not sure I get it, do you mean it in the way:
`allow_prereleases=False` -> call get_latest_release() in PyGithub
`allow_prereleases=True` -> workflow used in this PR?

Reply:
@lachmanfrantisek Ok, I will add it to the ogr, should I merge this anyway or wait for it?

Reply:
No need to wait, merge it... thanks